slamd/design_assistant/static/liquid.js

```diff
@@ -0,0 +1,101 @@
+import {
+  assignClickEventToOtherForm,
+  handleOtherSubmission,
+  handleAddingOther,
+} from "./other.js";
+import { assignClickEventToSubmitButton } from "./utils.js";
+
+function assignInputEventToLiquidForm() {
+  const custom_liquid_name_fields = document.querySelectorAll(
+    ".liquid_option_name"
+  );
+  custom_liquid_name_fields.forEach(function (custom_liquid_name_field) {
```

Comment:
Siehe Anmerkung im Teams. Wir könnten uns überlegen, nur ein weiteres Liquid zu unterstützen; dann sind die for loops usw. nicht mehr relevant. Können wir uns mal überlegen.